Domain/SecurityIdentityRetrievalStrategy.php

```diff
@@ -78,7 +79,7 @@ public function getSecurityIdentities(TokenInterface $token)
 
     private function isNotAuthenticated(TokenInterface $token): bool
     {
-        if (\defined('\Symfony\Component\Security\Core\Authorization\Voter\AuthenticatedVoter::PUBLIC_ACCESS')) {
+        if (interface_exists(CacheableVoterInterface::class)) {
```

Comment:
```suggestion
        if (method_exists($this->authenticationTrustProvider, 'isAuthenticated')) {
```

Let's make sure we don't have a wrong check and check the existence of the method we want to use 🙂

Reply:
Fixed!